src/nft/components/profile/view/ViewMyNftsAsset.tsx

```diff
@@ -29,11 +29,23 @@ const getNftDisplayComponent = (
 ) => {
   switch (assetMediaType) {
     case AssetMediaType.Image:
-      return <Card.Image />
+      return <Card.Image uniformAspectRatio={UniformAspectRatios.square} />
     case AssetMediaType.Video:
-      return <Card.Video shouldPlay={mediaShouldBePlaying} setCurrentTokenPlayingMedia={setCurrentTokenPlayingMedia} />
+      return (
+        <Card.Video
+          shouldPlay={mediaShouldBePlaying}
+          setCurrentTokenPlayingMedia={setCurrentTokenPlayingMedia}
+          uniformAspectRatio={UniformAspectRatios.square}
```

Comment:
you could probably make `uniformAspectRatio` optional with square as the default value